### PR TITLE
Add: 7 patterns

### PR DESCRIPTION
**Pull request by FabianCristancho**

### CODE REVIEW

File: source-web/static/images/backgrounds/visiwig.webp
Comment (cosmoart) [inline source-web/static/images/backgrounds/visiwig.webp:1]: This image does not look good in the 16/9 aspect ratio.

**Comment by cosmoart** [comment]
Thank you for your contribution! Here are some recommendations:

1. Ensure images look good in the 16:9 aspect ratio (1280x720 is recommended). You can use the devtools for precise screenshots.
2. Correct the "licencelink" property to "licenceLink", in camelCase.
3. Prioritize quality over quantity. Avoid resources like [Kandinsky](https://www.kandinsky.io) due to:
   - Poor design quality
   - Registration requirements
   - Very limited free plan (only 3 downloads per month)
   - Offers very few resources
4. Organize resources from best to worst (as much as possible). Resources like [svgbackgrounds](https://www.svgbackgrounds.com) should be placed last and like [Pattern Generator](https://doodad.dev/pattern-generator/) first.
5. When applicable, add a license description for clarity. For example, describe the CC BY 4.0 license as "Free with attribution."
6. To be more exact CSS3 Patterns Gallery has an MIT license.
7. According to [the terms of use of tartanify.com](https://tartanify.com/terms-of-use) there are several licenses, so it would be "Multiple" and not "Free".
8. You can optimize the images further, for example the patterns_gallery image can be reduced from 561 kB to 69.7 kB (-88%)
image


I know [CONTRIBUTING.md](https://github.com/cosmoart/free-resources/blob/main/CONTRIBUTING.md) doesn't have some of these recommendations, I'm working on it.

Thanks for your help!

**Comment by FabianCristancho** [comment]
ok @cosmoart thank you for your comments. I will work on this.

**Comment by FabianCristancho** [comment]
@cosmoart according to your comments, I uploaded changes that improve the section.

**Comment by cosmoart** [comment]
Here are some additional recommendations:

1. When providing a license description, please follow this format:
     ```json
     {
       "title": "Resource Title",
       "link": "Resource Link",
       "img": "Image Link",
       "license": "License Type",
       "licenceLink": "License Link",
       "licenceDescription": "Description (e.g., Free with attribution)"
     }
     ```
     So resources like Hero Patterns would be:
     ```json
     {
		"title": "Hero Patterns",
		"link": "https://www.heropatterns.com",
		"img": "/images/backgrounds/heropatterns.webp",
		"licence": "CC BY 4.0",
		"licenceLink": "https://heropatterns.com",
		"licenceDescription": "Free with attribution"
     }
     ```
     And Visiwig would be:
     ```json
     {
		"title": "Visiwig",
		"link": "https://www.visiwig.com/patterns/",
		"img": "/images/backgrounds/visiwig.webp",
		"licence": "Free",
		"licencelink": "https://www.visiwig.com/pattern-license/",
		"licenceDescription": "\"You can use the patterns in any project, commercial or personal without attribution or any costs, but...\""
       }
     ```

2. Remove the unused "kandinsky.webp" image.

3. Image Optimization:
   - Optimize the images further to reduce their size. Some images are excessively large, with one reaching 1.6MB!.
   - When taking screenshots with devtools, keep in mind that the resulting image has double the pixel dimensions of the original quality (e.g., 2560x1440 pixels). To reduce the image size you can use the Squoosh tool to resize images. Select 50% to make them 1280x720 pixels.

    image

    You can see that the weight of the images uploaded in the project does not exceed 100KB, so a good starting point would be to ensure that your images do not exceed that threshold.

    image

Feel free to reach out if you have any questions.

**Comment by FabianCristancho** [comment]
ok, i'll work on it!

**Pull request merged**